### PR TITLE
jakarta-ize-specification-document

### PR DESCRIPTION
**Pull request by ederks85**
Signed-off-by: Edwin Derks <ederks85@gmail.com>

### CODE REVIEW

**Comment by hussainnm** [comment]
Good work. Changes look good till now.

Could rebase your branch on master, so the pull request shows only your changes.

File: spec/src/main/asciidoc/core/EnterpriseBeanEnvironment.adoc [hunk 1]
```diff
@@ -3200,78 +3200,4 @@ concurrent access to the SessionContext object to be thread-safe.
...
-  env = ctx.getEnvironment();
- }
-
- public someBusinessMethod(...) ... {
-  String fooValue = env.getProperty("foo");
-  String barValue = env.getProperty("bar");
- }
- ...
-}
 ----
```
Comment (hussainnm) [inline spec/src/main/asciidoc/core/EnterpriseBeanEnvironment.adoc:3203]: Remove the last line. This is breaking the rest of the document.

Comment (ederks85) [reply spec/src/main/asciidoc/core/EnterpriseBeanEnvironment.adoc]: Thanks I had overlooked that

File: spec/src/main/asciidoc/core/EnterpriseBeanEnvironment.adoc [hunk 2]
```diff
@@ -3200,4 +3200,4 @@ concurrent access to the SessionContext object to be thread-safe.
 The container
 must follow the requirements for all other resources and configuration
 parameters specified in the Jakarta EE Platform specification <<a9861>>.
-----
+----
```
Comment (hussainnm) [inline spec/src/main/asciidoc/core/EnterpriseBeanEnvironment.adoc:3203]: I meant the line with ----, if you build the docs everything from Chapter 11 is broken.

Comment (ederks85) [reply spec/src/main/asciidoc/core/EnterpriseBeanEnvironment.adoc]: Yes I see it now. I've fixed it and validated that the spec document is now generated with Chapter 11 and further included

**Comment by hussainnm** [comment]
In the What's new section you need to mention the namespace transition from javax->jakarta.

**Comment by ederks85** [comment]
@hussainnm all your suggestions have been processed. If there's nothing else, this PR can be merged

**Comment by ederks85** [comment]
Still found a couple of EJB Container references

**Comment by tkburroughs** [comment]
@ederks85 Thanks, looks good, and now it has been merged.

**Pull request merged**